java/debugger.jpda.truffle/src/org/netbeans/modules/debugger/jpda/truffle/frames/models/DebuggingTruffleTreeModel.java

```diff
@@ -47,15 +50,15 @@
 import org.netbeans.spi.viewmodel.TreeModelFilter;
 import org.netbeans.spi.viewmodel.UnknownTypeException;
 
-@DebuggerServiceRegistration(path="netbeans-JPDASession/"+TruffleStrataProvider.TRUFFLE_STRATUM+"/DebuggingView",
```

Comment:
Introducing new path for Truffle? That forms an API and might be good to be documented somewhere - for example in `arch.xml` with `group="layer"`. But I understand that the content of `netbeans-JPDASession` registration hasn't been documented in great detail yet...

Reply:
Well, this is generally understood as a friend api. The user needs to be knowledgeable about how the individual services work and any changes need to take that into account.
This produces UI at the end, we need to have some flexibility in changing it as needed.